tests/gateway/test_razorpay.py

```diff
@@ -0,0 +1,148 @@
+from unittest.mock import patch
+
+import pytest
+from django.conf import settings
+
+from saleor.payment import ChargeStatus, PaymentError, TransactionKind
+from saleor.payment.gateways import razorpay
+from saleor.payment.gateways.razorpay.forms import (
+    RazorPayCheckoutWidget, RazorPaymentForm)
+
+
+@pytest.fixture()
+def mocked_gateway_client(razorpay_payment):
+    with patch('razorpay.Client') as mocked:
```

Comment:
Not a fan of fixtures with side effects.  Are there any other ways we can implement that?